scripts/test-infra/verify.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+# Install deps
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh| sh -s -- -b $(go env GOPATH)/bin v1.27.0
+export PATH=$PATH:$(go env GOPATH)/bin
+
+# Run verify steps
+make gofmt-verify -e
+make ci-lint -e
```

Comment:
Here we could also drop `-e`, I think

We don't have these targets in the Makefile? Just do `make verify` and add golangci-lint as a new make target that `verify` depends on?